internal/controller/apiextensions/claim/reconciler.go

```diff
@@ -501,6 +502,19 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		if kerrors.IsConflict(err) {
 			return reconcile.Result{Requeue: true}, nil
 		}
+		if err.Error() == errBindCompositeConflict {
```

Comment:
Sure, instead of creating a dedicated error type, I used k8s conflict error, IMHO it matches good here, but let me know if you disagree.

Reply:
Don't think that's a good idea. It's brittle. If somebody adds some client call in that func it might have unexpected consequences. Better be super precise. Conflicts are defined errors from an apiserver, borrowing them for client code is not good.

Reply:
thanks @sttts for the feedback. Added global err variable.